packages/relay/src/lib/eth.ts

```diff
@@ -1747,15 +1747,6 @@ export class EthImpl implements Eth {
     const fromAddress = await this.resolveEvmAddress(contractResult.from, requestIdPrefix, [constants.TYPE_ACCOUNT]);
     const toAddress = await this.resolveEvmAddress(contractResult.to, requestIdPrefix);
 
-    if (
```

Comment:
Q: Where the test being removed or added to affirm the change being made?
In a sense shouldn't an unset or `false` dev mode act exactly the same as this code being removed?